doc/reference.conf

```diff
@@ -1148,6 +1148,11 @@ general {
 	 */
 	kline_reason = "Connection closed";
 
+        /* SASL access only client message: give users a message that
+         * informs them
+         */
```

Comment:
This is indented with spaces; it must be indented with tabs to match the other lines.